x/programs/cmd/simulator/cmd/request.go

```diff
@@ -66,7 +66,7 @@ func (c *runCmd) Run(ctx context.Context, log logging.Logger, db *state.SimpleMu
 	if err = c.Verify(); err != nil {
 		return newResponse(0), err
```

Comment:
Should we not pass the `c.numRequests` here instead of 0 ? It's always safe to pass in the deref of the poniter to the value since it's initialized to 0 and only incremented in `RunRequest` (never set to nil) if I'm not mistaken.

Reply:
i dont think it matters since we are erroring here anyways and this is an empty response